scrapers/Iafd.yml

```diff
@@ -10,6 +15,18 @@ performerByURL:
     scraper: performerScraper
 
 xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h1/text()
+      Date: 
+        selector: //div[@class="col-xs-12 col-sm-3"]//p[text() = 'Release Date']/following-sibling::p[1]/text()
```

Comment:
`//div[@class="col-xs-12 col-sm-3"]//p[text() = 'Release Date']/following-sibling::p[1]/text()`
the trailing text `/text()` should be removed as it gives an extra empty date match  
`//div[@class="col-xs-12 col-sm-3"]//p[text() = 'Release Date']/following-sibling::p[1]` is enough

Reply:
this one is weird... I will test to make sure that you are right before changing